### PR TITLE
Display a logo and a background image in the header

### PR DESCRIPTION
**Pull request by nomad-alt**
### PT link: https://www.pivotaltracker.com/story/show/152637622 ###
- Add logo
- Add background image
- Add a header
- Add opening hours and order hours info

screenshot background

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -59,6 +68,31 @@ img.product_image {
   color: black;
 }
 
+.header{
+  width: 100%;
+  height: auto;
+  background-color: ;
```
Comment (olofvi) [inline src/app/app.component.css]: Maybe delete this line if it’s not in use

Comment (nomad-alt) [reply src/app/app.component.css]: Cleaned it up. 👍

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,60 +1,87 @@
...
+      <h5> Open: Monday - Friday 11-19</h5>
+      <h5> Order between: 11-18:30</h5>
+    </div>
+  </nav>
+  <br>
+  <br>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3">
+        <img class="sidebar_image"
```
Comment (Sajaas) [inline src/app/app.component.html]: I think we should rename it to sidedish_image or something like this. It's not a sidebar, and could confuse others. :)

Comment (olofvi) [reply src/app/app.component.html]: Are we not displaying opening hours during weekends? 
and linking to the about us page?

Comment (nomad-alt) [reply src/app/app.component.html]: This story is about logo and background. I'll fix open during weekend. @olofvi We can have a link when we create about us page. :)
`sidedish_image` 👍  @Sajaas

Comment (olofvi) [reply src/app/app.component.html]: just confusing because we were working on this story earlier https://www.pivotaltracker.com/story/show/152638119

**Pull request merged**